### PR TITLE
Minor fixes to esti workflow

### PR DESCRIPTION
**Pull request by eladlachmi**
Fix some minor issues with the esti workflow file.

### CODE REVIEW

File: .github/workflows/esti.yaml [hunk 1]
```diff
@@ -27,9 +27,21 @@ jobs:
         run: |
           echo "Enable next jobs based on secrets existence: ${{ env.ENABLE_NEXT_JOBS != '' }}"
           echo "secretsavailable=${{ env.ENABLE_NEXT_JOBS != '' }}" >> $GITHUB_OUTPUT
+  
+  extract-version:
```
Comment (N-o-Z) [inline .github/workflows/esti.yaml:31]: How do we extract version before we checkout the code?

File: .github/workflows/esti.yaml [hunk 2]
```diff
@@ -1007,6 +1009,10 @@ jobs:
       - name: Install dependencies
         working-directory: ./clients/python-wrapper
         run: pip install -r requirements.txt pytest pytest-md pytest-emoji
+
+      - name: Generate uniquifying value
```
Comment (N-o-Z) [inline .github/workflows/esti.yaml:1013]: Nice catch

**Review by N-o-Z — APPROVED** [review]
Thanks! One question that I don't understand

File: .github/workflows/esti.yaml (hunk 1 above)
Comment (eladlachmi) [reply .github/workflows/esti.yaml]: Magic 🎩🪄✨😊  
Fixed.

**Pull request merged**